sybil/sybil.py

```diff
@@ -1,11 +1,22 @@
 import inspect
 import sys
 from pathlib import Path
-from typing import Sequence, Callable, Collection, Mapping, Optional, Type, List
+from typing import Sequence, Callable, Collection, Mapping, Optional, Type, Dict, Any, List
+from unittest import TestSuite, TestLoader
 
 from .document import Document, PythonDocStringDocument, PythonDocument
 from .typing import Parser
 
+# Some of the types in this file are `_pytest.nodes.Collector` and
+# `.integration.pytest.SybilFile` types.
+# `.integration.pytest.SybilFile` depends on a `pytest` type.
+#
+# We do not want to require pytest as a runtime dependency for this file.
+# The ReadTheDocs build is done using the regular build dependencies.
+# Therefore, we need to use `Any` as the type for the pytest collector.
+_PytestNodesCollectorType = Any
+_IntegrationPytestSybilFileType = Any
```

Comment:
Why this rather than just the `Any` where they're used below? (I would ideally like to see this block go away)

Reply:
I can get rid of this and just use `Any` - I chose this because to me its a form of documentation for the developer.

Reply:
I don't think they add much, so please remove them.